libraries/resource_mysql2_chef_gem.rb

```diff
@@ -8,8 +8,9 @@ class Mysql2ChefGem < Chef::Resource::LWRPBase
       default_action :install
 
       attribute :mysql2_chef_gem_name, kind_of: String, name_attribute: true, required: true
-      attribute :gem_version, kind_of: String, default: '0.3.17'
+      attribute :gem_version, kind_of: String, default: '0.4.4'
       attribute :client_version, kind_of: String, default: nil
+      attribute :package_version, kind_of: String, default: nil
```

Comment:
Awesome. Thanks for the clarity. There should be corresponding README changes to reflect the new functionality.